### DISCUSSION TITLE
Include the Boostagram Specification in this Repository

### DISCUSSION THREAD

**Discussion by Alwin-Stockinger** [Podcastindex-org/podcast-namespace#506] [Enhancement Proposal]
I think the specification for Boostagrams should be moved to the podcast-namespace repository (instead of being managed in [https://github.com/lightning/blips/blob/master/blip-0010.md](https://github.com/lightning/blips/blob/master/blip-0010.md)). Discussions on enhancing or relating to it are already taking place in this repository, see for example:

- [https://github.com/Podcastindex-org/podcast-namespace/discussions/500](https://github.com/Podcastindex-org/podcast-namespace/discussions/500)
- [https://github.com/Podcastindex-org/podcast-namespace/discussions/466](https://github.com/Podcastindex-org/podcast-namespace/discussions/466)

Blip 10 (the current specification) suffers from duplicate fields (for example `time` and `ts`) and loosely explained fields such as `uuid`. I think iterating and improving the spec would be easier if Boostagrams would be defined in this repository.

There are also some (to my knowledge) unused fields such as `signature` and `sender_key` which could be removed if deemed unnecessary or reintroduced with a useful purpose.

If there is support for this I will make a Pull Request that includes the spec in this repository.

**Comment by daveajones** [comment]
Yes, please do this.  As soon as you do it I will propose the return address as a PR on this repo instead.

**Comment by genebean** [comment]
Sender key sounds sounds a lot like what @daveajones is proposing (just an observed bonus)

**Reply by Alwin-Stockinger** [reply]
Yes, but we would also need custom value/key to identify the sender completely and that would use up more space.
